Stripe/PublicHeaders/STPAuthenticationContext.h

```diff
@@ -34,7 +34,7 @@ NS_ASSUME_NONNULL_BEGIN
  
```

Comment:
call `completion`. I think we should call out that STPPaymentHandler will not proceed until completion is called